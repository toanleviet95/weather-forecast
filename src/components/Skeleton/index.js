import React from 'react';
import { Wrapper } from './index.styled';

const Skeleton = () => (
  <Wrapper>
    <div className='card'> 
      <div className='card__title loading' /> 
      <div className='card__description loading' /> 
      <div className='card__description loading' /> 
    </div> 
  </Wrapper>
);

export default Skeleton;
