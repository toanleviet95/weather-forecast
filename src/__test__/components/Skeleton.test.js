import React from 'react';
import renderer from 'react-test-renderer';
import Skeleton from '@/components/Skeleton';

describe('Skeleton', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Skeleton />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
