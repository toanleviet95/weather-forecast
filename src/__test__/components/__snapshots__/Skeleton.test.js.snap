// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton renders correctly 1`] = `
<div
  className="sc-bdnylx diXrbz"
>
  <div
    className="card"
  >
    <div
      className="card__title loading"
    />
    <div
      className="card__description loading"
    />
    <div
      className="card__description loading"
    />
  </div>
</div>
`;
